README.md

```diff
@@ -440,6 +440,34 @@ end
 
 jQuery and Handlebars are the main use cases for this flag.
 
+## Ruby support
+
+According to [these release notes][1.9.3-eol], Ruby versions prior to `2.0.x`
+has been end-of-lifed.
+
+Additionally, this codebase makes use of [(required) keyword arguments][kwargs].
+
+From `ember-cli-rails@0.4.0` and on, we will no longer support versions of Ruby
+prior to `2.1.0`.
+
+To use `ember-cli-rails` with older versions of Ruby, try the `0.3.x` series.
+
+[kwargs]: https://robots.thoughtbot.com/ruby-2-keyword-arguments
+[1.9.3-eol]: https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/
```

Comment:
I'd put the long explanation on something like NEWS file, or an UPGRADING guideline. In the README, I'd specify in one line what versions are required.